.bazelrc

```diff
@@ -37,9 +37,8 @@ build:remote-shared --remote_timeout=600
 build:remote-shared --remote_download_minimal
 build:remote-shared --jobs=100
 build:remote-shared --verbose_failures
-# TODO(sluongng): split these into separate configs for each target platform
-build:remote-shared --platforms=//platforms:linux_x86_64
-build:remote-shared --extra_execution_platforms=//platforms:linux_x86_64
+build:remote-shared --platforms=//platforms:local_config_platform
+build:remote-shared --extra_execution_platforms=//platforms:local_config_platform
```

Comment:
I don't think this should work for MacOS?

Effectively here, we were saying: if you gonna do a remote build, you _MUST_ be targeting linux x86_64. I set it up that way because:
1. Most of our executors are linux x86_64, we have a very limited number of macos executors
2. Our MacOS RBE does not work. There are a few problems with toolchains and Go SDK (see https://github.com/buildbuddy-io/buildbuddy/pull/5153)

In the future, I think we gonna do something like this
```
build:remote-shared --platforms=//platforms:linux_x86_64,//platforms:macos_arm64
build:remote-shared --extra_execution_platforms=//platforms:linux_x86_64,//platforms:macos_arm64
```

where we will be building multiple platforms remotely by default, instead of only a single platform that is compatible with the user's current host platform.

Reply:
kk, sounds good. Thanks for the explanation.